app/AppKernel.php

```diff
@@ -16,8 +16,8 @@ public function registerBundles()
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
-            new Fx\UserBundle\FxUserBundle(),
```

Comment:
If your final goal is to keep and reuse this UserBundle, it should live in a dedicated namespace like it was (Fx\UserBundle\FxUserBundle())

I was more confused about the ST\TricksBundle\STTricksBundle()

Reply:
Hello Olivier, thanks for your answer.
For more consistency in this project, i decided to leave it as is (without the Fx namespace). I imagine being a new person coming in the project and i would myself also ask me the goal of the Fx namespace (which would be a bit confusing), as you did for the ST namespace.
For future projects, and as you teached me how to do, this bundle will leave into the vendor dir.